packages/shared-base/src/Site/index.ts

```diff
@@ -4,6 +4,7 @@ import { ExtensionSite, EnhanceableSite } from './types.js'
 
 const matchEnhanceableSiteHost: Record<EnhanceableSite, RegExp> = {
     [EnhanceableSite.Localhost]: /localhost/i,
+    [EnhanceableSite.App]: /app\.mask\.io|netlify\.app|localhost/i,
```

Comment:
maybe remove netlify.app, it includes so many untrusted sites.

and please ensure match with hostname, not path. here is some example of evil URLs that can matches this expression:

- app.mask.io.evil.com
- anything.netlify.app
- localhost.evil.com
- evil.com/localhost
- app.mask.ioevil.com